### DISCUSSION TITLE
Currently Working on

### DISCUSSION THREAD

**Discussion by spandy1** [jaydeepukani/PJz6b5MjJ6dKm4KNH9QtqsrP#5] [General]
@spandy1  
- Instagram/FB Integration with Front end 
- Manage Multiple accounts from same platform (Instagram/FB Specific)

@jaydeepukani 
- Bugs Fixing in the current code 
- Facebook related Analytics

@hacker7599

- Analytics Graph/Table for SELF
- Analytics Graph/Table for Public

**Comment by kalrav9** [comment]
What bugs fixing? Please mention.

**Reply by jaydeepukani** [reply]
i think we can't call It's bug becuase we change our code to feature so in near future we can make features to paid features that's it

**Reply by kalrav9** [reply]
what features are we talking about here? What paid feature? Please don’t give a head topic on what things we are working. Mention every single detail/

**Reply by jaydeepukani** [reply]
i can't show you in discussion if you want to watch we can discuss on call and i will share my screen on anydesk

**Comment by kalrav9** [comment]
@jaydeepukani Any update?

**Reply by jaydeepukani** [reply]
yes in this week we changed lots of code so we commit code after once it's done

**Reply by kalrav9** [reply]
> yes in this week we changed lots of code so we commit code after once it's done

Yes, can you give day wise update since 16th July? What progress did we make? What bugs did we fix? What are we working on today?

**Reply by jaydeepukani** [reply]
today we are working on to change api from frontend to backend
